qutebrowser/html/dirbrowser.html

```diff
@@ -0,0 +1,68 @@
+{% extends "base.html" %}
+{% block style %}
+{{ super() }}
+#dirbrowserContainer {
+    background: #fff;
+    min-width: 35em;
+    max-width: 35em;
+    position: absolute;
+    top: 2em;
+    left: 1em;
+    padding: 10px;
+    border: 2px solid #eee;
+    -webkit-border-radius: 5px;
+}
+
+#dirbrowserTitleText {
+    font-size: 118%;
+    font-weight: bold;
+}
+
+ul {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+}
+
+ul > li {
+    background-repeat: no-repeat;
+    background-size: 22px;
+    line-height: 22px;
+    padding-left: 25px;
+}
+
+ul > li {
+    background-image: url('{{folder_icon}}');
+}
+
+ul.files > li {
+    background-image: url('{{file_icon}}');
+}
+{% endblock %}
+
+{% block content %}
+    <div id="dirbrowserContainer">
+        <div id="dirbrowserTitle">
+            <p id="dirbrowserTitleText">Browse directory: {{url}}</p>
+        </div>
+
+        <ul>
```

Comment:
Can you add a `class="parent"` or so here? This will make it easier to write the tests.